modules/lb/variables.tf

```diff
@@ -31,6 +31,12 @@ variable "domain" {
   description = "this is used to find an existing issued wildcard cert and route53 zone"
 }
 
+variable "private_zone" {
+  type        = "string"
+  description = "this is used to if route53 zone is private or not"
```

Comment:
Could you change this to `Set to true if route53 zone is private`?

Reply:
Updated it